WordPress/wordpress-playground
# Support ?mode=browser-full-screen for a full-sized browser window with no colorful background

## What is this PR doing?

Adds a new Query API parameter `?mode=browser-full-screen` to display Playground settings control but without the blank space around the Playground site. It's like `?mode=seamless` plus Playground UI controls. It's particularly useful for:

* Designing and testing themes, when you need both Playground settings and the real estate on the screen
* Embedding Playground on other sites without the unnecessary padding while retaining the ability to change the settings

### mode=browser

<img width="1591" alt="CleanShot 2023-11-28 at 14 04 50@2x" src="https://github.com/WordPress/wordpress-playground/assets/205419/43743d3a-0cc3-4801-88d4-d534b8643f74">

### mode=browser-full-screen

<img width="1588" alt="CleanShot 2023-11-28 at 14 04 58@2x" src="https://github.com/WordPress/wordpress-playground/assets/205419/62e9ddd9-0aed-4511-8206-d182ec8b9c6c">


## Testing Instructions

* Go to http://localhost:5400/website-server/ and confirm it's the good old browser UI like on the first screenshot above
* Go to http://localhost:5400/website-server/?mode=browser-full-screen and confirm it looks like on the second screenshot above
* Go to http://localhost:5400/website-server/?mode=seamless and confirm it's just a fullscreen WordPress website without any UI controls around it
